src/quo2/components/avatars/channel_avatar/view.cljs

```diff
@@ -9,13 +9,14 @@
 
 (defn- initials
   [full-name size color]
-  (let [amount-initials (if (= size :size/l) 2 1)]
+  (let [amount-initials (if (= size :size/l) 2 1)
+        channel-name    (string/replace full-name #"[# ]" "")]
```

Comment:
hmm, is initials specific to a channel or is this generic to all avatars? perhaps a different word for this variable name is better if it's generic

Reply:
Hi @J-Son89, Thank you very much for reviewing PR. They are generic to all channel-avatars.
![image](https://github.com/status-im/status-mobile/assets/17097240/91607e1a-11a3-41ef-86e8-0f90aab71d61)

> perhaps a different word for this variable name is better if it's generic

Any suggestions :)

Reply:
Oh if it's only for channel avatars then it seems fine. I thought it might be used for account, user etc

Reply:
Replaced regex with simple string ("#"). They both supposed to work similar in our case (channel-avatars are not supposed to have spaces in their name), still this looks better choise.